website/source/docs/providers/aws/d/redshift_account_id.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: aws_redshift_account_id"
+sidebar_current: "docs-aws-datasource-redshift-account-id"
+description: |-
+  Get AWS Redshift Account ID
+---
+
+# aws\_redshift\_account\_id
+
+Use this data source to get the Account ID of the [AWS Redshift Account](http://docs.aws.amazon.com/redshift/latest/mgmt/db-auditing.html#db-auditing-enable-logging)
+in a given region for the purpose of negotiating with other AWS services.
```

Comment:
I'd suggest we make the description a bit more specific than this since there are two lists of accounts IDs related to Redshift. The current description could also (confusingly) match http://docs.aws.amazon.com/redshift/latest/mgmt/db-auditing.html#rs-db-auditing-cloud-trail-rs-acct-ids - i.e. account IDs of accounts that perform calls to other AWS services on behalf of the user.